### PR TITLE
Add in /tickets links for RSVPs and tickets

### PR DESCRIPTION
**Pull request by nealfennimore**
https://central.tri.be/issues/115994

### CODE REVIEW

**Comment by juanfra** [comment]
Good job! What I meant was having methods on:

1. `plugins/tickets/src/Tribe/Blocks/Rsvp.hp`
1. `plugins/tickets/src/Tribe/Blocks/Tickets.php`

Like "get_tickets_link()" with the logic that there is right now on `order-links.php` before printing and calling them on the `render()` method of each class, sending them as a variable to the template. Then if there's a message of "you have X tickets" it is printed, else, the template is not included.

Maybe @mitogh has other idea to implement this.

File: plugins/tickets/src/views/blocks/attendees/order-links.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+if ( $has_rsvps ) {
+	$counters[] = sprintf( _n( '%d %s', '%d %ss', $rsvp_count, 'event-tickets' ), $rsvp_count, $ticket_type );
+}
+
+if ( $has_tickets ) {
+	$counters[] = sprintf( _n( '%d %s', '%d %ss', $ticket_count, 'event-tickets' ), $ticket_count, $ticket_type );
+}
+
+$link = $view->get_tickets_page_url( $event_id, $is_event_page );
+$message  = sprintf( esc_html__( 'You have %s for this %s.', 'event-tickets' ), implode( __( ' and ', 'event-tickets' ), $counters ), $events_label_singular );
```
[the remark this answers is not in the document]
Comment (nealfennimore) [reply plugins/tickets/src/views/blocks/attendees/order-links.php]: Think I got this, can you check again?

File: plugins/tickets/src/views/blocks/attendees/order-links.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+$has_rsvps = $is_rsvp && 0 !== $rsvp_count;
+$has_tickets = $is_ticket && 0 !== $ticket_count;
+
+if ( $has_rsvps ) {
+	$counters[] = sprintf( _n( '%d %s', '%d %ss', $rsvp_count, 'event-tickets' ), $rsvp_count, $ticket_type );
+}
+
+if ( $has_tickets ) {
+	$counters[] = sprintf( _n( '%d %s', '%d %ss', $ticket_count, 'event-tickets' ), $ticket_count, $ticket_type );
```
[the remark this answers is not in the document]
Comment (nealfennimore) [reply plugins/tickets/src/views/blocks/attendees/order-links.php]: I'm not entirely sure what you mean on this one, but I moved everything down below the template guard

**Comment by mitogh** [comment]
Requires @bordoni approval.

**Pull request merged**